Exercise_A/index.html

```diff
@@ -1,45 +1,270 @@
+  <!--Allows us to track the progress of the scroll -->
+  <!-- Nav bar items -->
+  <nav id="mainNav" class="navbar navbar-default navbar-expand-md fixed-top">
+    <div class="container-fluid ">
+      <div class="navbar-header ">
+        <a class="navbar-brand page-scroll" href="#top">The Only Bootstrap Page</a>
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarScroll"
+          aria-controls="navbarScroll" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+      </div>
 
+      <div class="collapse navbar-collapse justify-content-end" id="navbarScroll">
+        <ul class="nav navbar-nav nav-pills">
+          <li class="nav-item"><a class="page-scroll" href="#about">About</a> </li>
+          <li class="nav-item"><a class="page-scroll" href="#service">Four Icons</a> </li>
+          <li class="nav-item"><a class="page-scroll" href="#portfolio">Bullshit</a> </li>
+          <li class="nav-item"><a class="page-scroll" href="#contact">Call Me</a> </li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+  <!-- Can use Jumbotron -->
+  
+    <header>
+      <div class=" header-content">
+        <div class="header-content-inner">
+          <h1 style="font-size: 60.5833px;">Hey Look, It's Every Bootstrap Website Ever</h1>
+          <hr>
+          <p>Take a look around at the same fucking bootstrap page you've seen ten million times before! Featuring all
+            the
+            same things as those pages!</p>
+          <a href="#about" class="btn btn-1 btn-lg btn-primary page-scroll">This button makes the page scroll</a>
+        </div>
+      </div>
+    </header>
+    <!-- Second Section -->
+    <section id="about" class="orange">
+      <div class="container">
+        <div class="row">
+          <div class="col-lg-8 offset-lg-2 text-center">
+            <h2 class="section-heading">Want to make an original website yourself?</h2>
+            <hr class="light">
+            <p class="text-faded">Forget that! Who would ever want to put in all of that effort for a website? Just open
+              up your web browser and type "bootstrap template" into your favorite search engine, like Yahoo! or Bing,
+              and
+              you're on your way! There are hundreds of templates to choose from, but go ahead and pick this same exact
+              template from the first result on google, edit a few lines, and you're on your way! No one will notice!
+            </p>
+            <a href="#" target="_blank" class="btn btn-default page-scroll btn-lg">Google That Shit</a>
+          </div>
+        </div>
+      </div>
+    </section>
     <!-- Four icons -->
+    <section id="service">
+      <div class="container">
+        <div class="row">
+          <div class="col-lg-6 textcenter">
+            <h2 class="section-heading">So Fucking Creative</h2>

```

Comment:
You should probably just remove the col-lg-6 from the div above as it is forcing this title to just be o the left.